draft-ietf-idr-linklocal-capability.xml

```diff
@@ -50,14 +50,26 @@
     <area>Routing</area>
     <workgroup>Interdomain Routing</workgroup>
     <abstract>
-      <t>BGP <xref target="RFC4271" format="default"/>, was originally designed
-to provide reachability between domains and between the edges of a domain.
-As such, BGP assumes the next hop towards any reachable destination may
-not reside on the advertising speaker, but rather may either be through
-a router connected to the same subnet as the speaker, or through a router
-only reachable by traversing multiple hops through the network. Because
-of this, as per <xref target="RFC4291" format="default"/> - BGP does not recognize  IPv6 link-local addresses,
-as a valid next hop for the forwarding purposes.</t>
+      <t>
+        BGP <xref target="RFC4271" format="default"/>, was originally designed
```

Comment:
Is the format element providing much for us here?  I think you can skip it.

Reply:
Removed it from every xref.